aml-development/frontend
# fix(style): fix firefox header style

To test:
Use Firefox before pulling branch.  Notice the header is shifted.
Pull branch, header is fixed.

Also makes sure the menu dropdown is on top of the header, instead of behind it.
